c

```diff
@@ -1,4 +1,4 @@
-BUILDH="yes"
```

Comment:
I am willing to approve and merge this - but could you please comment on why it was necessary to unset this variable?